assets/scss/modular/_resource-search.scss

```diff
@@ -0,0 +1,51 @@
+
+/*
+
+---
+name: Search
+category:
+ - Components/Resources
+---
+
+```html
+<div class="resource-search__container">
+	<svg><use xlink:href="#icon-search"></svg>
+	<input type="search" placeholder="Search for resources" class="resource-search">
+</div>
+```
+
+*/
+
+.resource-search__container {
+	position: relative;
+	color: color('mid-grey');
+}
+
+.resource-search__container > svg {
```

Comment:
Yes, it prevents issues occurring with Firefox, that see's the SVG within the shadow dom too.